src/node/build.ts

```diff
@@ -118,6 +125,7 @@ export async function build({ script = 'sync', mock = false } = {}) {
   )
 
   await fs.remove(ssgOut)
+  await fs.remove(join(out, 'manifest.json')
```

Comment:
Could we have manifest for ssg build and copy it over instead of populating the users' output?

Reply:
Ok, maybe later I don't have my computer now.

Reply:
Did you mean use other name?

Reply:
> Did you mean use other name?

That will do too. 

> maybe later I don't have my computer now.

No hurries, take your time :)

Reply:
After generate client, move manifest to ssgOut and read manifest from ssgOut, finally delete ssgOut. Is that ok?

Reply:
My concern is that if users have the manifest.json under their `public/` this would override it. Do you have any idea on solving this?